test/app_web/controllers/edit_count_controller_test.exs

```diff
@@ -1,20 +1,29 @@
 defmodule AppWeb.EditCountControllerTest do
   use AppWeb.ConnCase
-  alias App.{Issue, Repo}
+  alias App.{Issue, User, Repo}
 
   describe "get svg count for an issue" do
     setup do
+      user = %{
+        login: "user_login",
+        user_id: 1,
+        avatar_url: "/avatar.jpg",
+        html_url: "/user/1"
+      }
+      user_changeset = User.changeset(%User{}, user)
+      user = Repo.insert!(user_changeset)
```

Comment:
Why not write this as a nice pipe ([how phoenix defaults to writing this pattern in the generators](https://github.com/phoenixframework/phoenix/blob/b68fdbadfda871d29bff1e6368c1447a103a48c5/priv/templates/phx.gen.context/schema_access.ex#L48)):

```elixir
user =
  %User{
    login: "user_login",
    user_id: 1,
    avatar_url: "/avatar.jpg",
    html_url: "/user/1"
  }
  |> User.changeset
  |> Repo.insert!
```